HtmlBuilderCore/Elements/HtmlTable.cs

```diff
@@ -0,0 +1,70 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace HtmlBuilderCore.Elements
+{
+    public class HtmlTable : IHtmlElement
+    {
+
+        public int rows { get; set; }
+        public int columns { get; set; }
+        public string values { get; set; }
+        public List<object> list { get; set; }
+        int i, j;
+
+        public HtmlTable()
+        {
+            list = new List<object>();
+        }
+        public string getTable = "";
+        public void AddRow(params string[] rowdata)
+        {
```

Comment:
So you've made the AddRow more a function like AddTableData where you take that one row and fill the entire table with it.

As a side note, this is why the naming makes a BIG difference. We can talk about this design later.

Nothing to change here for now. We can just have a discussion about this.